approval-decision-utils-product/README.md

```diff
@@ -0,0 +1,237 @@
+
+# Approval Decision Utils
+  
+This component provides a standardized approach for implementing the approval flow. Users can make decisions, confirmations, leave comments, and view the approval history in a data table.
+  
+## Demo
+  
+###  1. Display decision option
+![](./images/1-request.PNG)
+
+###  2. Validate decision option
+![](./images/3-request-validate.PNG)
+
+###  3. Track approval history
+![](./images/4-request-history.PNG)
+
+###  4. Select checkbox confirmation
+![](./images/5-request-confirmation.PNG)
+
+## Setup
+  
+###  Approach  
+  
+Decision data is stored in database table named `ApprovalHistory`. This table stores the selected decision, comment, approval date and confirmations.
```

Comment:
Just to be coherent:
Decision data is stored in a database table named `ApprovalHistory.` This table stores the selected decisions, comments, approval dates, and confirmations.